tools/fix_auth/fix_auth.rb

```diff
@@ -84,17 +84,17 @@ def load_rails
 
     def set_passwords
       ManageIQ::Password.key_root = cert_dir if cert_dir
-      if options[:legacy_key]
-        if (legacy_key = ManageIQ::Password.load_key_file(options[:legacy_key]))
-          ManageIQ::Password.keys["alt"] = legacy_key
-        else
-          puts "WARNING: key #{options[:legacy_key]} not found"
-        end
-      end
+    end
+
+    def set_legacy_key
+      key = ManageIQ::Password.load_key_file(options[:legacy_key])
+      puts "WARNING: legacy key #{options[:legacy_key]} not found" unless key
+      options[:legacy_key] = key
```

Comment:
this will still set `legacy_key` if the file was not found. The previous code did not.
Is this a change that we want?

Reply:
ignore this
Not common use case and even less of a common error case. I think it will probably work well even if the user really screws up